### DISCUSSION TITLE
Feature: Sort apps into their respective categories

### DISCUSSION THREAD

**Discussion by keilmillerjr** [stuarthayhurst/alphabetical-grid-extension#62] [Ideas]
Thank you for alphabetically sorting apps. One step further would be automatically sorting apps into their respective categories. Hand sorting is useless. It's just like android. A bunch of icons that leads to frustration and using the search anyways.

Related links for reference:
https://github.com/BenJetson/gnome-dash-fix
https://gitlab.gnome.org/GNOME/gnome-shell-extensions

**Comment by stuarthayhurst** [comment]
I'm not against the idea, but I'm also not going to implement this myself, as I mostly plan on just maintaining this project, with no new features for the foreseeable future. If someone submits a PR to do this, I'm happy to help when I can, and I'll merge it when everything is up to scratch.
